### PR TITLE
PUD-106 - Split to two tests & fix create recall test

### PR DESCRIPTION
**Pull request by jonnywyatt**

### CODE REVIEW

File: src/test/java/cucumber/pages/BookRecallPage.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package cucumber.pages;
+
+import net.serenitybdd.screenplay.targets.Target;
+import net.thucydides.core.annotations.DefaultUrl;
+
+@DefaultUrl("https://manage-recalls-dev.hmpps.service.justice.gov.uk/assess-recall")
```
Comment (AndyNewton) [inline src/test/java/cucumber/pages/BookRecallPage.java:6]: Future: I'm not sure what @DefaultUrl is bringing us but should we have multiple test page classes with the same value for this?

File: src/test/java/cucumber/pages/RecallRecommendationPage.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package cucumber.pages;
+
+import net.serenitybdd.screenplay.targets.Target;
+import net.thucydides.core.annotations.DefaultUrl;
+
+@DefaultUrl("https://manage-recalls-dev.hmpps.service.justice.gov.uk/assess-recall")
```
Comment (AndyNewton) [inline src/test/java/cucumber/pages/RecallRecommendationPage.java:6]: Ditto.

File: src/test/java/cucumber/pages/UploadRecallDocumentsPage.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package cucumber.pages;
+
+import net.serenitybdd.screenplay.targets.Target;
+import net.thucydides.core.annotations.DefaultUrl;
+import net.serenitybdd.core.pages.WebElementFacade;
+import org.openqa.selenium.WebDriver;
+import net.thucydides.core.pages.components.FileToUpload;
+import org.openqa.selenium.By;
+
+@DefaultUrl("https://manage-recalls-dev.hmpps.service.justice.gov.uk/assess-recall")
```
Comment (AndyNewton) [inline src/test/java/cucumber/pages/UploadRecallDocumentsPage.java:10]: Ditto.

File: src/test/resources/features/managerecalls/ManageRecall.feature [hunk 4]
```diff
@@ -1,20 +1,27 @@
...
+    And Maria clicks on the Create recall button
+    Then Maria continues from the Book a recall page
+    And Maria recommends a 14 day recall
+    And Maria continues from the Upload documents page
+    Then Maria is on the Assess Recall page
+    And Maria confirms the recall length as 14 days
+    When Maria clicks on the download revocation order link
+    Then a revocation order is downloaded
+
+  Scenario: Can navigate to assess a recall from the To do list
```
Comment (AndyNewton) [inline src/test/resources/features/managerecalls/ManageRecall.feature:21]: Split of the Scenario is fine with me: I know we merged two to one a couple of weeks ago but I'm not a fan of mega-scenarios.

File: src/test/resources/features/managerecalls/ManageRecall.feature [hunk 5]
```diff
@@ -1,20 +1,27 @@
...
+    Given Maria navigates to manage recall service
+    When Maria logs in
     And Maria clicks To Do
     Then Maria is on the ToDo Recalls page
-    #FIXME - this should use the ID of the recall that we just created
```
Comment (AndyNewton) [inline src/test/resources/features/managerecalls/ManageRecall.feature:15]: Why remove the comment and the following check?  It's still the case that this depends on a prior created recall existing, at least.

**Review by AndyNewton — APPROVED** [review]
Approving but please eye-ball comments before merging.  Thanks.

File: src/test/resources/features/managerecalls/ManageRecall.feature (hunk 5 above)
Comment (jonnywyatt) [reply src/test/resources/features/managerecalls/ManageRecall.feature:15]: Restored

File: src/test/resources/features/managerecalls/ManageRecall.feature (hunk 4 above)
Comment (jonnywyatt) [reply src/test/resources/features/managerecalls/ManageRecall.feature:22]: 👍

File: src/test/java/cucumber/pages/BookRecallPage.java (hunk 1 above)
Comment (jonnywyatt) [reply src/test/java/cucumber/pages/BookRecallPage.java]: Removed the dupes

**Pull request merged**